app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -179,14 +195,21 @@ function AcuantCapture(
         <FullScreen onRequestClose={() => setIsCapturingEnvironment(false)}>
           <AcuantCaptureCanvas
             onImageCaptureSuccess={(nextCapture) => {
+              let result;
               if (nextCapture.glare < ACCEPTABLE_GLARE_SCORE) {
                 setOwnErrorMessage(t('errors.doc_auth.photo_glare'));
+                result = 'glare';
               } else if (nextCapture.sharpness < ACCEPTABLE_SHARPNESS_SCORE) {
                 setOwnErrorMessage(t('errors.doc_auth.photo_blurry'));
+                result = 'blurry';
               } else {
                 onChangeAndResetError(nextCapture.image.data);
+                result = 'success';
               }
 
+              const agent = /** @type {NewRelicGlobal} */ (window).newrelic;
+              agent?.addPageAction('documentCapture.acuantResult', { result });
```

Comment:
WDYT of calling this `acuantSdkResult` or something to indicate it's from the client SDK and not the acuant backend? Or is that implied since this is a JS event?

Reply:
> WDYT of calling this `acuantSdkResult` or something to indicate it's from the client SDK and not the acuant backend? Or is that implied since this is a JS event?

It could be implied since `addPageAction` is part of NewRelic's browser API, but I don't think it hurts to be extra detailed here that it's coming from the Acuant web SDK.

Reply:
Updated in aff504b7fe4208863da12ec4797db795c98add44.